WooCommerce/UITestsFoundation/Screens/Orders/UnifiedOrderScreen.swift

```diff
@@ -236,4 +236,26 @@ public final class UnifiedOrderScreen: ScreenObject {
 
         return self
     }
+
+    /// Checks the screen for existence of all products, checking each name.
+    /// - Parameter productNames: Array of product names to check.
+    /// - Returns: Unified Order screen object.
+    @discardableResult
```

Comment:
```suggestion
```

think we can remove the `@discardableResult` since this has a return value that we know and want to use.